scripts/pack.py

```
@@ -0,0 +1,325 @@
+    return nav
+
+
+# ── Generate build configs ────────────────────────────────────────────────────
+def generate_build_configs():
+    import yaml
+
+    cfg = yaml.safe_load(Path("mkdocs.yml").read_text(encoding="utf-8"))
+
+    nav = auto_generate_nav(cfg.get("docs_dir", "docs"))
+    print(
+        f"  Auto-generated nav with {sum(len(v) if isinstance(v, dict) and isinstance(list(v.values())[0], list) else 1 for v in nav)} page(s)"
```

Ruff (0.15.21)

[warning] Prefer `next(iter(v.values()))` over single element slice

Replace with `next(iter(v.values()))`

(RUF015)